src/xdocs/config_regexp.xml

```diff
@@ -1104,7 +1104,13 @@ void method() {
   &lt;property name=&quot;ignoreComments&quot; value=&quot;true&quot;/&gt;
 &lt;/module&gt;
         </source>
-
+        <p>Example:</p>
```

Comment:
We need example if default config. It does nothing, so please explain in words above it that it will catch nothing. No code sample is required for such default.

Please add config/examples for all properties that Check has.